src/looper/index.ts

```diff
@@ -157,7 +157,6 @@ function loopProbe({ probe, notifications }: loopProbeParams) {
   const flags = getContext().flags
   // flags is the monika parameter flags... these are used:
```

Comment:
I think we dont need these comments anymore. we can cleanup and erase this?

Reply:
Sure, I'll remove them.